test/scripts/create-tax-integration.test.js

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
Something really picky noticed, the file name is better to be with `s`, `create-tax-integrations`